packages/web-components/src/checkbox/checkbox.ts

```diff
@@ -32,58 +26,72 @@ export class BaseCheckbox extends FASTElement {
    */
   public get checked(): boolean {
     Observable.track(this, 'checked');
-    return this._checked;
+    return !!this._checked;
   }
 
-  public set checked(state: boolean) {
-    this._checked = state;
+  public set checked(next: boolean) {
+    const prev = this._checked;
+    this._checked = next;
+
+    if (!!prev !== !!next) {
+      Observable.notify(this, 'checked');
+    }
```

Comment:
Should this be something like this to avoid executing the code below?

```ts
if (!!next === this._checked) {
  return;
}
this._checked = next;
Observable.notify(this, 'checked');
```

Reply:
It turns out `Observable.notify()` shouldn't be guarded in getters.